src/stories/SearchInput.story.js

```diff
@@ -6,7 +6,7 @@ import { Row, StoryItem } from './utils.js';
 
 import SearchInput from '../lib/SearchInput';
 
-const stories = storiesOf('Core', module);
+const stories = storiesOf('Core', module); // eslint-disable-line no-undef
```

Comment:
Is it about `module` ? You can put globals in the config or alternatively an `env: node` in the config for stories directory.